### PR TITLE
Task dev 3

### PR DESCRIPTION
**Pull request by HannaKisel**

### CODE REVIEW

File: DEV-3/SequenceFib/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bin/
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/.gitignore]: add gitignore for all repository, not only for this project

File: DEV-3/SequenceFib/SequenceFib/Program.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace SequenceFib
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.Write("Please, enter number ");
+            String inputString = Console.ReadLine();
+            int numbFibonacci = int.Parse(inputString);
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/SequenceFib/Program.cs]: Can throw exception, use try-catch

File: DEV-3/SequenceFib/SequenceFib/Program.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            if (numbFibonacci < 0)
+            {
+                Console.WriteLine("I can't check this number");
+            }
+            int[] array = new int[numbFibonacci + 3];
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/SequenceFib/Program.cs]: Why do you memorize all Fibonacci numbers in massive? use two temporal variables

File: DEV-3/SequenceFib/SequenceFib/Program.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                catch (Exception)
+                {
+                    Console.WriteLine("It is not a number. Try again");
+                    continue;
+                }
+                work = false;
+            }
+        }
+    }
+}
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/SequenceFib/Program.cs]: Remove extra linrs

Comment (HannaKisel) [reply DEV-3/SequenceFib/SequenceFib/Program.cs]: Which liners?

File: DEV-3/SequenceFib/SequenceFib.sln [hunk 5]
```diff
@@ -0,0 +1,22 @@
+﻿
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/SequenceFib.sln]: One solution for all projects, no need to create a separate one

File: DEV-3/SequenceFib/SequenceFib/Program.cs [hunk 6]
```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace SequenceFib
+{
+    class Program
+    {
+        static void Main()
+        {
+            bool work = true;
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/SequenceFib/Program.cs]: Bad variable name

File: DEV-3/SequenceFib/SequenceFib/Program.cs [hunk 7]
```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace SequenceFib
+{
+    class Program
```
Comment (IgorKheidorov) [inline DEV-3/SequenceFib/SequenceFib/Program.cs]: Bad class name

**Review by IgorKheidorov — APPROVED** [review]
Not bad but not perfect

**Pull request merged**